modules/ingestor/src/graph/sbom/clearly_defined.rs

```diff
@@ -18,40 +16,40 @@ impl SbomContext {
         let mut purls = PurlCreator::new();
         let mut licenses = LicenseCreator::new();
 
-        let mut assertions = Vec::new();
+        // TODO: Since the node id cannot be obtained here, it’s not possible to replace purl_license_assertion with sbom_package_license.
```

Comment:
It's ok to keep a TODO. But I'd appreciate if we could add a tracking issue for this.

Reply:
https://github.com/trustification/trustify/issues/1419